__tests__/api.test.js

```diff
@@ -219,3 +219,39 @@ describe("/POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/PATCH /api/articles/:article_id", () => {
+  const updateVotes = {
+    inc_votes: 10,
+  };
+  it("status:200, responds with updated article", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send(updateVotes)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(5);
+        expect(response.body.article.votes).toBe(10);
+      });
+  });
+  it("status: 404, responds with an appropriate status and error when given a valid but non-exisitant article_id", () => {
+    return request(app)
+      .patch("/api/articles/104")
+      .send(updateVotes)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe("No such article_id exist yet");
+      });
+  });
+  it("status: 400, responds with an appropriate status and error when given invalid datatype for votes", () => {
```

Comment:
Where is your 400 test for an invalid article_id?

Reply:
Sorry, I had the 404 test for invalid article_id. Has been using 404 for invalid article_ids in previous tests as well.